BeVolt/App/src/main.c

```diff
@@ -74,17 +83,37 @@ int realmain(){
  *		- Give wrappers (Voltage, Current, Temperature) the limits
  */
 void initialize(void){
-	PLL_Init80MHz();
 	LED_Init();
 	Contactor_Init();
 	Contactor_Off();
 	WDTimer_Init();
 	EEPROM_Init();
 	SoC_Init();
-
 	Current_Init();
 	Voltage_Init(Minions);
 	Temperature_Init(Minions);
+	CLI_Init(Minions);
+
+	fifoInit(&CLIFifo);
+	__enable_irq();
+	CLI_Startup();
+
+	// Checks to see if the batteries need to be charged
+	Voltage_UpdateMeasurements();
+	SafetyStatus voltage = Voltage_CheckStatus();
+	if(voltage == UNDERVOLTAGE) {
+		printf("Do you need to charge the batteries? (y/n)\n\r>> ");
```

Comment:
When should we look into this?

Reply:
We should probably get Matthew and Billy in on this soon. If we all have time this Saturday, we could talk about it then. It may be as simple as a message from the BPS giving permission to enable the motor, but the implementation details are going to be important.